sql/branch.sql

```diff
@@ -0,0 +1,29 @@
+-- Alter the enums to cater for missing data.
+ALTER TYPE payment_net ADD VALUE 'unknown';
+ALTER TYPE exchange_status ADD VALUE 'unknown';
+
+-- Update the field status in the exchanges table from NULL to 'unknown'
+UPDATE exchanges SET status = 'unknown' WHERE status IS NULL;
+
+-- Alter the exchanges table to ensure that no more NULL values are entered
+ALTER TABLE exchanges ALTER COLUMN status SET NOT NULL;
+
+-- Insert records for ‘unknown’ (previously NULL in exchanges table
+-- network in exchange_route table
+INSERT INTO  exchange_routes (participant, network, address, error)
+     (
+       SELECT DISTINCT participants.id, 'unknown'::payment_net, 'None', 'None'
```

Comment:
Goofy to use the string "None" here. I think the empty string might be a better choice?